diff --git a/CMakeLists.txt b/CMakeLists.txt
index 67153d2..d7fc19b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -131,21 +131,25 @@ endif()
 
 add_subdirectory(libdssp)
 
-add_executable(mkdssp ${CMAKE_CURRENT_SOURCE_DIR}/src/mkdssp.cpp)
+option(DSSP_BUILD_MKDSSP "Build the mkdssp executable" ON)
 
-target_link_libraries(mkdssp PRIVATE libmcfp::libmcfp dssp::dssp)
+if(DSSP_BUILD_MKDSSP)
+	add_executable(mkdssp ${CMAKE_CURRENT_SOURCE_DIR}/src/mkdssp.cpp)
 
-if(USE_RSRC)
-	mrc_target_resources(mkdssp
-		${CIFPP_SHARE_DIR}/mmcif_pdbx.dic
-		${CIFPP_SHARE_DIR}/mmcif_ddl.dic
-		${CMAKE_CURRENT_SOURCE_DIR}/libdssp/mmcif_pdbx/dssp-extension.dic)
-endif()
+	target_link_libraries(mkdssp PRIVATE libmcfp::libmcfp dssp::dssp)
+
+	if(USE_RSRC)
+		mrc_target_resources(mkdssp
+			${CIFPP_SHARE_DIR}/mmcif_pdbx.dic
+			${CIFPP_SHARE_DIR}/mmcif_ddl.dic
+			${CMAKE_CURRENT_SOURCE_DIR}/libdssp/mmcif_pdbx/dssp-extension.dic)
+	endif()
 
-# Install rules
-install(TARGETS ${PROJECT_NAME}
-	RUNTIME DESTINATION bin
-)
+	# Install rules
+	install(TARGETS ${PROJECT_NAME}
+		RUNTIME DESTINATION bin
+	)
+endif()
 
 if(BUILD_DOCUMENTATION)
 	add_subdirectory(doc)
